src/main.ts

```diff
@@ -11,6 +11,7 @@ function setupSwagger(app: INestApplication) {
     .setTitle(process.env.APP_SWAGGER_Title ?? "Cophr")
     .setDescription(process.env.APP_SWAGGER_Description ?? "")
     .setVersion(process.env.APP_SWAGGER_Version ?? "N/A")
+    .addBearerAuth()
```

Comment:
Seems the docs need more infos here. The docs will resulted in error

Reply:
Revised
[refactor: add swagger securitySchemes](https://github.com/Cophr/backend/pull/10/commits/f30d2b940a8ded14f2bcb6a2d06cb21852331fab)